test/templates/confA/a.conf

```diff
@@ -0,0 +1,2 @@
+test_1
```

Comment:
nitpick: There is convention in Golang to store test fixtures in `testadata` directory, either on root level or in each package directory.

https://tutorialedge.net/golang/advanced-go-testing-tutorial/#use-the-testdata-directory
https://eli.thegreenplace.net/2022/file-driven-testing-in-go/

Reply:
To be honest, this file was generated automatically (I don't know by what), but anything in the `test` directory is not created by me. I will exclude this from PR, sorry for misleading.